pkg/api/server/v1alpha2/results_test.go

```diff
@@ -147,6 +149,112 @@ func TestGetResult(t *testing.T) {
 	}
 }
 
+func TestUpdateResult(t *testing.T) {
+	srv, err := New(test.NewDB(t))
+	if err != nil {
+		t.Fatalf("failed to create server: %v", err)
+	}
+	ctx := context.Background()
+
+	tt := []struct {
+		name      string
+		parent    string
+		in        *pb.Result
```

Comment:
`in` isn't used meaningfully - AFAICT only to set unique names. We can generate pseudo-random names from the loop index instead.

Reply:
Done